product_listing/product_listing.html

```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home | Key Lorem</title>
+    <link rel="stylesheet" href="https://monad-ui.netlify.app/componentLibrary.css">
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="/product_listing/product_listing.css">
+</head>
+<body class="body-container">
+     
+    <nav class="navbar">
+        <h3 class="nav__heading">Lorem</h3>
+        <div class="search-container">
+            <input type="search" name="searchBar" class="input__search" placeholder="search here">
+            <a href="#"><button class="btn__search"><img src="/images/home-page/search.png" alt="search" class="img--icon"></button></a>
```

Comment:
img tag is self-closing, please use / before closing img tag otherwise it will cause problems